src/iperf_sctp.c

```diff
@@ -525,8 +545,10 @@ iperf_sctp_bindx(struct iperf_test *test, int s, int is_server)
     }
 
     if (sctp_bindx(s, xaddrs, nxaddrs, SCTP_BINDX_ADD_ADDR) == -1) {
+        saved_errno = errno;
```

Comment:
We need a declaration of `int saved_errno` at the top of this function (`iperf_sctp_bindx()`) or compilation blows up here (and again three lines lower).

Reply:
Looks like I missed one, sorry about that.

Reply:
No worries, thanks for fixing!  Admittedly I could have just gone and fixed this myself post-merge, but I'm also using this to learn about the GitHub pull request review workflow, which is still new to me.

Reply:
Verified last change after installing the sctp packages on Linux.

Reply:
It also fixes the build on FreeBSD.  Looks like we're good to go.  Thanks again!